github-profiles/NicoovLamperouge-893/NicoovLamperouge#893.md5

```diff
@@ -0,0 +1,37 @@
+# Hi, i'm Nicolas 
+
+Front end Jr
```

Comment:
I recommend to put "frontend web developer Jr". Frontend it's a single word. And then, it is recommended to put a complete description, to know that it is web development.